charts/metrics-server/values.yaml

```
@@ -76,6 +76,8 @@ hostNetwork:
 
 replicas: 1
 
+revisionHistoryLimit: 10
```

```suggestion
revisionHistoryLimit:
```
We shouldn't be setting anything to a default value.